src/fragments/04-headings-hr-tags.scss

```diff
@@ -142,7 +157,8 @@ body:not(.ebullientworks-nix-tags) {
     --tag-color: var(--color-accent-2);
     --tag-color-hover: var(--tag-3);
   }
-  &.theme-dark .print,
+
+  &.theme-dark,
```

Comment:
There was a distinction here, as the settings applied only to the dark theme when printing: `&.theme-dark .print`